src/site/content/en/live/index.njk

```diff
@@ -123,32 +123,33 @@ date: 2020-06-31
 
   <hr />
 
-  {% if event.isPostEvent %}
-  <section id="map" class="w-event-section">
-
-    <div class="w-event-section__column-left">
-      <h3 class="w-mt--non">
-        <span class="w-event-icon w-event-icon__map"></span>
-        Regional events
-      </h3>
-      <p>
-        Missed web.dev LIVE? We’ve got you covered. Thanks to our Google Developer Group communities, we have a ton of live events.
-      </p>
-      <a href="/live#content" class="w-event-link">Back to top</a>
-    </div>
+  {% if true %}
+  {# {% if event.isPostEvent %} #}
+    <section id="map" class="w-event-section">
+
+      <div class="w-event-section__column-left">
+        <h3 class="w-mt--non">
+          <svg class="w-event-icon" width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-.61.08-1.21.21-1.78L8.99 15v1c0 1.1.9 2 2 2v1.93C7.06 19.43 4 16.07 4 12zm11.99 4c.9 0 1.64.59 1.9 1.4A7.991 7.991 0 0020 12c0-3.35-2.08-6.23-5.01-7.41V5c0 1.1-.9 2-2 2h-2v2c0 .55-.45 1-1 1h-2v2h6c.55 0 1 .45 1 1v3h1z" fill="#5F6368"/></svg>
```

Comment:
just double checking — did you run the svg through svgomg?

Reply:
yes, it shrank it a tiny bit